unswear.user.js

```diff
@@ -2,19 +2,19 @@
 // @name         devrant-unswear
 // @namespace    http://tampermonkey.net/
 // @version      0.5
-// @description  remove bad words from post
+// @description  Remove bad words from any post
 // @author       devTeaa
 // @match        https://devrant.com/*
 // @grant        none
 // ==/UserScript==
 
 (function() {
-  // Change this to false if you want specific badWords to be replace with their goodWords
-  // but the goodWords must have the same lenght as badWords
+  // Change randomMode to false if you want specific badWords to be replaced with their goodWords.
+  // If randomMode's false, goodWords must have the same length as badWords. 
   let randomMode = true;
 
   let goodWords = ["❤️", "🎀"];
-  let badWords = ["fuck"];
+  let badWords = ["fuck(\w+)?", "motherfuck(\w+)?", "shit(\w+)?", "bitch(\w)?"];
```

Comment:
Do we really need the (\w+)? as I didn't see much difference on it
```suggestion
  let badWords = ["fuck", "motherfuck", "shit", "bitch"];
```

Reply:
With "(\w+)?", regex can match words like "fucking", "fuckity", "motherfucking" and-the-like. It means something like "Match 'fuck'. The word can contain optional characters after the last character -- In this case, 'k' --." This way, we can filter more bad words without hesitating too much (thanks to Regex ^^). I hope I could explain myself (I'm not pretty good at it. lol).

Reply:
Oh that makes more sense, I didn't notice the changes Thanks for the PR